gridsome/lib/webpack/plugins/corejsBabelPlugin.js

```diff
@@ -0,0 +1,13 @@
+const corejsPath = require('path').dirname(require.resolve('core-js/package.json')).replace(/\\/g, '/')
+
+module.exports = () => ({
+  visitor: {
+    ImportDeclaration(path) {
+      if (path.node.specifiers.length !== 0 || path.node.source.value.indexOf('core-js/') !== 0) {
+        return
+      }
+
+      path.node.source.value = corejsPath + path.node.source.value.substr('core-js'.length)
+    }
+  }
+})
```

Comment:
This should be replaced with https://github.com/babel/babel-polyfills/tree/main/packages/babel-plugin-polyfill-corejs3 as doing this naive find and replace can break stuff really easily.

Unsure which version of `@babel/core` or `@babel/preset-env` is needed for it to work (cc @nicolo-ribaudo) so if you'd like to update the babel dependencies in a seperate commit that would probably be best